sematic/db/models/artifact.py

```diff
@@ -6,10 +6,11 @@
 
 # Sematic
 from sematic.db.models.base import Base
+from sematic.db.models.mixins.has_organization_mixin import HasOrganizationMixin
 from sematic.db.models.mixins.json_encodable_mixin import JSON_KEY, JSONEncodableMixin
 
 
-class Artifact(Base, JSONEncodableMixin):
+class Artifact(HasOrganizationMixin, Base, JSONEncodableMixin):
```

Comment:
Let's add mixins after `Base`, as the order of precedence for method resolution is left to right. Though there (hopefully) shouldn't be any method name conflicts between our mixins and SQLAlchemy `Base`, it is more of a proper subclassing and less of a mixin, so it should likely take precedence.

Reply:
Done.